lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,112 @@
+/*https://codepen.io/Digitango/pen/YQQBXR pulsing animation source code */
```

Comment:
💯  citing your source makes you a good community member, and helps with debugging if there's ever a problem in borrowed code.

Reply:
This isn't still in progress - lesson 4-6 have been submitted on the course site as well. Is there something I was missing that I should complete on this lesson that made it incomplete?

Reply:
Also there is another transformation I would like to use and I'm struggling with it - would you be able to have some office hours with me?  :)